src/components/Filters/components/Input/Input.tsx

```diff
@@ -42,13 +43,23 @@ export const StyledInput = styled.input`
   }
 `;
 
-const Input: React.FC<InputProps> = ({ value = '', onChange }) => (
-  <StyledInput placeholder="String" value={value} onChange={onChange} />
+const Input: React.FC<InputProps> = ({
+  value = '',
+  onChange,
+  onKeyDown = noop,
```

Comment:
why do we need `onKeyDown`

Reply:
Found it, in this case, it's maybe better to use destructured rest of props via `{...props}` instead of picking one of many properties

Reply:
I am not sure I understand where the destructing would happen here

Reply:
In this `Input` component
```
const Input: React.FC<InputProps> = ({
  value = '',
  onChange,
  ...props
}) => (
  <StyledInput
    placeholder="String"
    value={value}
    onChange={onChange}
    {...props}
  />
);